.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  arrowParens: "avoid",
+  bracketSameLine: true,
+  bracketSpacing: true,
+  jsxSingleQuote: true,
+  objectCurlySpacing: true,
+  printWidth: 100,
+  singleQuote: true,
```

Comment:
`singleQuote: true,` yet this is written with double quotes 😓 
in your package json add a yarn script called "prettier:fix" that does `npx prettier --write ./` and run it right after

Reply:
done